src/Domain/Wallet.cs

```diff
@@ -0,0 +1,256 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using dotnetstandard_bip39;
```

Comment:
At this moment, I think we'd better not depend on that nuget package, but fork the code instead. @ccorcoveanu, @AdoAdoAdo, how do you feel about this?

In erdpy, we've simply re-written the functions, to match the ones from https://github.com/alepop etc.:

https://github.com/ElrondNetwork/elrond-sdk-erdpy/blob/main/erdpy/wallet/core.py